### ISSUE TITLE
Error when creating a new organization

### ISSUE THREAD

**Issue by niryariv** [okfn/docker-ckan#26]
When trying to create a new org on a fresh install the response is a 500 Error, 
`OSError: [Errno 13] Permission denied: '/var/lib/ckan/storage/uploads`

Seems like the issue is caused by `/var/lib/ckan/storage` being owned by root. Opening a terminal in the container and changing owner to ckan fixed it.

**Comment by SamuelBradley** [comment]
Couple of things to add, I had this bug too but only in the production deployment (dev setup was not affected), my fix was to add:

```
# Fix bug https://github.com/okfn/docker-ckan/issues/26
RUN mkdir -p /var/lib/ckan/storage/uploads
RUN chown -R ckan:ckan /var/lib/ckan/storage
```

to ckan/Dockerfile

FYI this path is in a docker volume so this fix will only work on a fresh deployment or after deleting the CKAN_STORAGE volume. Which shouldn't be an issue because you won't get far before running into this bug

**Comment by amercader** [comment]
This was fixed by #46
